reflex/components/core/layout/center.py

```diff
@@ -0,0 +1,24 @@
+"""A center component."""
+
+from reflex.components.component import Component
+from reflex.components.el.elements.typography import Div
+from reflex.style import Style
+
+
+class Center(Div):
+    """A center component."""
+
+    tag = "div"
```

Comment:
does this even need to be set if you inherit from `Div`?

Reply:
no i'll remove it.